libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -193,22 +238,22 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
     }
   }
 
-  private getFeatureFlagValue(
+  private getFeatureFlagValueFromV1Schema(
```

Comment:
The word V1Schema is confusion-leading because the MS FM schema also has v1 version. how about call it `LegacySchema` ?

Reply:
And call the new schema with its official name Microsoft FeatureManagement schema, that's `MsFmSchema` in code

Reply:
Makes sense. This has been renamed